<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Trn_DelayWidget</class>
 <widget class="QWidget" name="Trn_DelayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="spinBox6">
   <property name="geometry">
    <rect>
     <x>337</x>
     <y>260</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>70</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Net-Out 8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox9">
   <property name="geometry">
    <rect>
     <x>532</x>
     <y>260</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
  </widget>
  <widget class="QLabel" name="label18">
   <property name="geometry">
    <rect>
     <x>1115</x>
     <y>70</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Net-Out 8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label10">
   <property name="geometry">
    <rect>
     <x>595</x>
     <y>70</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Net-Out 8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox18">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>260</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
  </widget>
  <widget class="QLabel" name="label6">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>70</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Net-Out 8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox17">
   <property name="geometry">
    <rect>
     <x>1055</x>
     <y>260</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox5">
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>260</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
  </widget>
  <widget class="QSlider" name="slider9">
   <property name="geometry">
    <rect>
     <x>545</x>
     <y>94</y>
     <width>19</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>55</number>
   </property>
  </widget>
  <widget class="QLabel" name="label5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Net-Out 8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="slider5">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>93</y>
     <width>20</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>55</number>
   </property>
  </widget>
  <widget class="QSlider" name="slider10">
   <property name="geometry">
    <rect>
     <x>613</x>
     <y>94</y>
     <width>19</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>55</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox13">
   <property name="geometry">
    <rect>
     <x>795</x>
     <y>260</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox4">
   <property name="geometry">
    <rect>
     <x>207</x>
     <y>260</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox14">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>260</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox3">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>260</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
  </widget>
  <widget class="QLabel" name="label8">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>70</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Net-Out 8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="slider17">
   <property name="geometry">
    <rect>
     <x>1068</x>
     <y>94</y>
     <width>19</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>55</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox2">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>260</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox11">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>260</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox20">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>260</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox1">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>260</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSlider" name="slider19">
   <property name="geometry">
    <rect>
     <x>1198</x>
     <y>94</y>
     <width>19</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>55</number>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>70</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Net-Out 8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox8">
   <property name="geometry">
    <rect>
     <x>467</x>
     <y>260</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox15">
   <property name="geometry">
    <rect>
     <x>925</x>
     <y>260</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
  </widget>
  <widget class="QLabel" name="label17">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>70</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Net-Out 8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label20">
   <property name="geometry">
    <rect>
     <x>1245</x>
     <y>70</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Net-Out 8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox7">
   <property name="geometry">
    <rect>
     <x>402</x>
     <y>260</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
  </widget>
  <widget class="QLabel" name="label15">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>70</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Net-Out 8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label9">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Net-Out 8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox10">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>260</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
  </widget>
  <widget class="QSlider" name="slider2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>94</y>
     <width>19</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>55</number>
   </property>
  </widget>
  <widget class="QSlider" name="slider14">
   <property name="geometry">
    <rect>
     <x>873</x>
     <y>94</y>
     <width>19</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>55</number>
   </property>
  </widget>
  <widget class="QSlider" name="slider15">
   <property name="geometry">
    <rect>
     <x>938</x>
     <y>94</y>
     <width>19</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>55</number>
   </property>
  </widget>
  <widget class="QSlider" name="slider6">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>94</y>
     <width>19</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>55</number>
   </property>
  </widget>
  <widget class="QSlider" name="slider8">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>94</y>
     <width>19</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>55</number>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>In Delay Control</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="slider3">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>94</y>
     <width>19</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>55</number>
   </property>
  </widget>
  <widget class="QLabel" name="label11">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>70</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Net-Out 8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox19">
   <property name="geometry">
    <rect>
     <x>1185</x>
     <y>260</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
  </widget>
  <widget class="QSlider" name="slider20">
   <property name="geometry">
    <rect>
     <x>1263</x>
     <y>94</y>
     <width>19</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>55</number>
   </property>
  </widget>
  <widget class="QSlider" name="slider18">
   <property name="geometry">
    <rect>
     <x>1133</x>
     <y>94</y>
     <width>19</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>55</number>
   </property>
  </widget>
  <widget class="QSlider" name="slider1">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>94</y>
     <width>19</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>55</number>
   </property>
  </widget>
  <widget class="QLabel" name="label13">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>70</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Net-Out 8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label16">
   <property name="geometry">
    <rect>
     <x>985</x>
     <y>70</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Net-Out 8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox16">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>260</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
  </widget>
  <widget class="QLabel" name="label19">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>70</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Net-Out 8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="slider4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>94</y>
     <width>19</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>55</number>
   </property>
  </widget>
  <widget class="QLabel" name="label14">
   <property name="geometry">
    <rect>
     <x>855</x>
     <y>70</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Net-Out 8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="slider11">
   <property name="geometry">
    <rect>
     <x>678</x>
     <y>94</y>
     <width>19</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>55</number>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Net-Out 8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label12">
   <property name="geometry">
    <rect>
     <x>725</x>
     <y>70</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Net-Out 8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox12">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>260</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
  </widget>
  <widget class="QSlider" name="slider16">
   <property name="geometry">
    <rect>
     <x>1003</x>
     <y>94</y>
     <width>19</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>55</number>
   </property>
  </widget>
  <widget class="QSlider" name="slider13">
   <property name="geometry">
    <rect>
     <x>808</x>
     <y>94</y>
     <width>19</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>55</number>
   </property>
  </widget>
  <widget class="QSlider" name="slider12">
   <property name="geometry">
    <rect>
     <x>743</x>
     <y>94</y>
     <width>19</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>55</number>
   </property>
  </widget>
  <widget class="QSlider" name="slider7">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>94</y>
     <width>19</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>55</number>
   </property>
  </widget>
  <widget class="QLabel" name="label7">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Net-Out 8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Net-Out 8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
